test/helpers.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+import "@total-typescript/ts-reset";
```

Comment:
I don't see it being used in code 🤔

Reply:
it's an override for builtin typescript types

e.g. JSON.parse normally returns `any` but it should return `unknown` - this import makes it return unknown among other things

Reply:
got it, thx